ytelse-ung/src/main/java/no/nav/ung/sak/ytelse/ung/beregning/UngdomsytelseGrunnlagBuilder.java

```diff
@@ -17,29 +19,33 @@ class UngdomsytelseGrunnlagBuilder {
     private static final Logger log = LoggerFactory.getLogger(UngdomsytelseGrunnlagBuilder.class);
 
     private LocalDateTimeline<UngdomsytelseSatser> perioder = LocalDateTimeline.empty();
+    private Clob satsBeregningRegelInput;
+    private Clob satsBeregningRegelSporing;
     private UngdomsytelseUttakPerioder uttakPerioder;
 
     private boolean built = false;
 
     UngdomsytelseGrunnlagBuilder(UngdomsytelseGrunnlag kladd) {
         if (kladd != null && kladd.getSatsPerioder() != null) {
-            this.leggTilPerioder(kladd.getSatsPerioder().getPerioder());
+            var satsPerioder = kladd.getSatsPerioder();
+            this.leggTilPerioder(satsPerioder.getPerioder(), satsPerioder.getRegelInput().getClob(), satsPerioder.getRegelSporing().getClob());
             this.uttakPerioder = kladd.getUttakPerioder();
         }
     }
 
-    UngdomsytelseGrunnlagBuilder leggTilPerioder(List<UngdomsytelseSatsPeriode> perioder) {
+    UngdomsytelseGrunnlagBuilder leggTilPerioder(List<UngdomsytelseSatsPeriode> perioder, Clob regelInput, Clob regelSporing) {
```

Comment:
Det kunne kanskje vært best å bruke String på utsiden av entiteten?

Reply:
Ja, enig. Det var for å slippe å ha så mange lob'er, men eg tenkjer vi heller kopierer heile perioder-aggregatet